src/app/Login.tsx

```diff
@@ -0,0 +1,22 @@
+"use client"
+import { useSession, signIn, signOut } from "next-auth/react";
+import { UserCard } from "./userCard";
+
+export default function Login() {
+    const { data: session } = useSession();
+    console.log(session);
+    if(session) { //if user logged in, show signout
+        return (
+            <>
+                <button onClick={() => signOut()} type="button" className="btn btn-primary">Sign Out of Google</button>
+                <UserCard user={session?.user}/>
+            </>
+        )
+    } else { //if not logged in, sign in
+        return (
+            <>
+                <button onClick={() => signIn()} type="button" className="btn btn-primary">Sign In with Google</button>
```

Comment:
Please add the 'google' parameter to the `signIn()` function so we skip the intermediate page and go straight to the Google sign in!

Reply:
@kaseyliu @Bdon101 Make sure you handle this small change if you have time and I'll approve your pull request!